tests/unit/src/main/scala/tests/CodeLensesTextEdits.scala

```diff
@@ -0,0 +1,23 @@
+package tests
+
+import java.util
+import org.eclipse.lsp4j.TextEdit
+import org.eclipse.{lsp4j => l}
+import scala.collection.JavaConverters._
+
+object CodeLensesTextEdits {
+  def apply(ranges: util.List[l.CodeLens]): List[l.TextEdit] = {
```

Comment:
Let's use `Seq[l.CodeLens]` and force the call-site to use `.asScala` if necessary. By convention, I try to use Scala collections/futures in our internal APIs